colossalai/utils/model/lazy_init_context.py

```diff
@@ -40,10 +40,12 @@ class LazyInitContext():
             to value setting, such as `zero_` and `triu_`. `zero_` is pre-added by default.
     """
 
-    tensor_set_value_func = ['zero_']
+    tensor_set_value_func = ['zero_', 'fill_']
```

Comment:
updates the Note of the descriptions above?

Reply:
Sure.